cmd/bpf2go/gen/output.go

```diff
@@ -90,10 +90,15 @@ type GenerateArgs struct {
 	ObjectFile string
 	// Output to write template to.
 	Output io.Writer
+	// Function which transforms the input into a valid go identifier (internal.Identifier if left nil)
```

Comment:
Note: referring to internal symbols in godoc is a bit strange, since they don't appear in the godoc UI. Something like 'uses the default behaviour if nil' is sufficient here.